components/common/src/ui.rs

```diff
@@ -53,21 +53,14 @@ impl From<Color> for termcolor::Color {
     }
 }
 
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, Default)]
 pub enum GlyphStyle {
+    #[default]
```

Comment:
seems like the default woul be full for both platforms now whereas it was limited for windows.